src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4849,4 +4849,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ElseClauseRequiredWithWhenClause" xml:space="preserve">
     <value>An 'else' clause is required when a 'when' clause is present.</value>
   </data>
-</root>
+  <data name="ERR_SemiOrLBraceOrArrowExpected" xml:space="preserve">
+    <value>{ or ; or =&gt; expected</value>
```

Comment:
I am not sure what point you are trying to make. What I meant to say is that tokens are usually quoted in error messages so that it is clear that they are language tokens rather than a punctuation in the sentence.